### PR TITLE
Fix/catalog azure server 2020 

### PR DESCRIPTION
**Pull request by jpitapeva**
## Hey, I just made a Pull Request!

issue https://github.com/backstage/backstage/issues/11679

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by jpitapeva** [comment]
hi, @jhaals and @awanlin 

I opened this new PR and now i learned to user commit -s :)

**Comment by jpitapeva** [comment]
old PR below with problems with DCO
https://github.com/backstage/backstage/pull/11701

File: .changeset/shy-geese-judge.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': patch
+---
+
+Updated to support deployments of Azure DevOps Server under TFS or similar subpath
```
Comment (jhaals) [inline .changeset/shy-geese-judge.md:5]: Vale warns about the spelling of subpath, it should probably be updated to sub path instead. Don't forget to sign the commit with `-s` :)

**Review by benjdlambert — APPROVED** [review]

**Comment by jpitapeva** [comment]
@benjdlambert, @awanlin , Hi, i have one question, do you know when this PR enter for install using 'npx @backstage/create-app' (version)

Thanks

**Comment by awanlin** [comment]
@jpitapeva took a quick look and it was released in the `next` release path last Tuesday: https://github.com/backstage/backstage/blob/master/docs/releases/v1.3.0-next.2-changelog.md#backstageintegration121-next2

I think that the `main` release path will come out Tuesday June 21, but I may have my dates wrong

**Comment by jpitapeva** [comment]
> @jpitapeva took a quick look and it was released in the `next` release path last Tuesday: https://github.com/backstage/backstage/blob/master/docs/releases/v1.3.0-next.2-changelog.md#backstageintegration121-next2
> 
> I think that the `main` release path will come out Tuesday June 21, but I may have my dates wrong

Thanks

**Comment by awanlin** [comment]
@jpitapeva looks like I was wrong, the `main` release was today, and this change is in it. 

https://github.com/backstage/backstage/releases/tag/v1.3.0

Thanks again for contributing this improvement!

**Comment by benjdlambert** [comment]
👍  thanks again! as @awanlin mentioned, it should be in the latest release as of today!

**Comment by jpitapeva** [comment]
> 👍 thanks again! as @awanlin mentioned, it should be in the latest release as of today!

Thank u

**Comment by jpitapeva** [comment]
> @jpitapeva looks like I was wrong, the `main` release was today, and this change is in it.
> 
> https://github.com/backstage/backstage/releases/tag/v1.3.0
> 
> Thanks again for contributing this improvement!


Thank u

**Pull request merged**